data/mods/blazemod/items/guns/blob_turret.json

```diff
@@ -0,0 +1,351 @@
+[
```

Comment:
Simple one here, just pulled all of the `"type": "GUN"` entries from blaze_blob.json and stuck them here for better sorting.